### ISSUE TITLE
Hysteresis not working as intended on Aourus RTX 3070

### ISSUE THREAD

**Issue by MaxGamingLV** [Rem0o/FanControl.Releases#1901]
**Describe the bug**
[A clear and concise description of what the bug is.  Screenshots are worth a thousand words. Hardware support requests do NOT count as a bug. See https://github.com/Rem0o/FanControl.Releases#issues-and-hardware-compatibility.]

The issue that i've been having is the GPU going up and down in fan speed even though my hysteresis is set to 4 degrees celcius and i've also set it to only apply on the way down in temperature! No matter what i do, it doesn't wait until it has passed the 4 degree threshold. It just changes the fan speed right as it gets to the lower point in the curve.
For example, i'm playing a lighter 3D game like minecraft and the GPU gets to 49 degrees, it goes up from 50 to 53% GPU fan speed but when it goes below the 49 degrees to 48, it goes back to 50% which makes no sense as i have setup hysteresis...

**Is there a log.txt file next to FanControl.exe with recent date entries?**
[Link it here]

**Relevant hardware specs and setup**
[Laptops and most proprietary pre-built desktops are not supported.]

My Gaming Desktop PC Specifications. [DIY Built]

Intel Core i5-12600k
RTX 3070 Aorus Master
32GB DDR5 5600MHz
Gigabyte Aorus Z690 Elite AX DDR5 mobo
Corsair RM850 PSU
NZXT Kraken X53 Cooler
2x top exhaust NZXT Fans via Top Radiator
1x rear exhaust Arctic P12
3x front intake Arctic P12 Max
Case Fractal Design Meshify C
Windows 11 Home

![IMG_5902](https://github.com/Rem0o/FanControl.Releases/assets/45124410/95076956-6952-45c4-9158-0e8d32fc2015)

**Comment by Rem0o** [comment]
The most left (idle) and right (load) points are ignored for hysteresis. This is done for the idle/load points to be reachable with a high hysteresis value.

EG: Your last temperature change which triggered a fan % change was 49 degree, and your lowest point in your graph is 48. You have let say a 7 degree hysteresis. You would need (49-7) = 42 degree to get a change, but your card might not even ever hit below 45, so you would be stuck at that point and never hit your idle point.  

However, hysteresis will apply when going from your lowest set point to a higher value ( going out of idle ).

TLDR: Your graph will snap to your idle point when hit, but hysteresis will apply to get out of that idle point.

**Comment by MaxGamingLV** [comment]
So essentially, by adjusting my first point further back to let's say 20 degrees would fix the hysteresis issues?

Point 1 at 20 degrees 0%
Point 2 at 21 degrees 50%
Point 3 at 48 degrees 53%

With hysteresis set to 4 degrees only on the way down, would it work?
When at 53%, it hits 46 at doesn't go down to 50% but waits for 44 degrees.

**Comment by Rem0o** [comment]
all you need is one more point at let say, (20C, 50%)

**Comment by MaxGamingLV** [comment]
Thank you for your fast answers and help! I've donated a bit of money to this project of yours and will continue to support it as it goes into the future!
